.github/workflows/workflows.yml

```diff
@@ -0,0 +1,43 @@
+name: Test workflows
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Setup engines
+      run: |
+        curl -o requirements.txt "https://raw.githubusercontent.com/dockstore/dockstore/1.13.0-beta.3/dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt"
+        pip3 install --user -r requirements.txt
+        
+    - name: Setup CLI 
+      run: | 
+        curl -L -o dockstore https://github.com/dockstore/dockstore/releases/download/1.12.2/dockstore
+        chmod +x dockstore       
+        mkdir /home/runner/dockstore_bin
+        mv dockstore /home/runner/dockstore_bin
+        echo "/home/runner/dockstore_bin" >> $GITHUB_PATH
+    - name: Prime CLI
+      run: | 
+        echo "$PATH"
+        mkdir ~/.dockstore
+        # we recommend suppressing INFO output for Cromwell, but if you run into a problem
+        # you may need to change ERROR to INFO to give you hints on how to debug
+        printf "cromwell-vm-options: -DLOG_LEVEL=ERROR\nserver-url: https://dockstore.org/api" > ~/.dockstore/config
+        dockstore
+
+    - name: Run cwl workflows with CLI
+      run: |
+        bash cli-script.sh
+
+    - name: Run wdl workflow with CLI
+      run: |
+        dockstore --script  workflow launch --local-entry checker.wdl  --json md5sum.wdl.json
```

Comment:
Regardless of whether or not cli-script.sh should exist or not, "Run cwl workflows with CLI" as is runs `set -eux -o pipefail` before running the Dockstore CLI, but my understanding "Run wdl workflow with CLI" 's Dockstore call does not operate under these options. Whether we keep cli-script.sh or not, I think `set -eux -o pipefail` needs to be added to the WDL context.

Reply:
Removing script to be consistent